src/rust-crypto/backup.ts

```diff
@@ -342,20 +342,29 @@ export class RustBackupManager extends TypedEventEmitter<RustBackupCryptoEvents,
 
                 if (!request || this.stopped || !this.activeBackupVersion) {
                     logger.log(`Backup: Ending loop for version ${this.activeBackupVersion}.`);
+                    if (!request) {
+                        // nothing more to upload
+                        this.emit(CryptoEvent.KeyBackupSessionsRemaining, 0);
```

Comment:
Some test relies on this event to be fired at the end of the backup loop.

Reply:
this comment should probably be in the code, not the PR.